docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,279 @@
+    #. Refine the symmetry of the relaxed structure, and find a standardised primitive cell using SeeK-path_
+    #. Run a self-consistent field calculation on the refined structure
+    #. Run a band structure calculation at fixed Kohn-Sham potential along a standard path between high-symmetry k-points determined by SeeK-path_
+
+The workflow uses the PBE exchange-correlation functional with suitable pseudopotentials and energy cutoffs from the `SSSP library version 1.1 <https://www.materialscloud.org/discover/sssp/table/efficiency>`_.
+
+.. _SeeK-path: https://www.materialscloud.org/work/tools/seekpath
+
+.. K-point mesh is selected to have a minimum k-point density of 0.2 Å-1
+   A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations
+
+The workflow should take ~10 minutes on your virtual machine.
+You may notice that ``verdi process list`` now shows more than one entry.
+While you wait for the workflow to complete, let's start exploring its provenance.
+
+The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+
+Start the AiiDA REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+
+and open the |provenance browser| (from the browser inside the virtual machine).
+
+.. |provenance browser| raw:: html
+
+    <a href="https://www.materialscloud.org/explore/ownrestapi?base_url=http://127.0.0.1:5000/api/v3" target="_blank">Materials Cloud provenance browser</a>
+
+.. note::
+
+    The provenance browser is a Javascript application that connects to the AiiDA REST API.
+    Your data never leaves your computer.
+
+.. some general comment on importance of the graph?
+.. a sentence on how to continue from here
+
+Browse your AiiDA database:
+
+    * Start by finding your Quantum ESPRESSO calculation (the type of node is called a ``CalcJobNode`` in AiiDA, since it is run as a job on a scheduler).
+    * Select ``Calculations`` in the left menu to filter for calculations only.
+    * Inspect the raw inputs and outputs of the calculation, and use the provenance browser to explore the input and output nodes of the calculation and the whole provenance of your simulations.
+
+.. note::
+
+    When perfoming calculations for a publication, you can export your provenance graph using ``verdi export create`` and upload it to the `Materials Cloud Archive <https://archive.materialscloud.org/>`_, enabling your peers to explore the provenance of your calculations online.
+
+Once the workchain is finished, use ``verdi process show <PK>`` to inspect the ``PwBandStructureWorkChain`` and find the PK of its ``band_structure`` output.
+Use this to produce a PDF of the band structure:
+
+.. code-block:: console
+
+   $ verdi data bands export --format mpl_pdf --output band_structure.pdf <PK>
+
+.. figure:: include/images/si_bands.png
+   :width: 80%
```

Comment:
Maybe we should update this figure, my band structure ended up being similar but slightly different (the relevant part I think would that the reference zero of energy is moved)

Reply:
Good to know! Thanks!